### PR TITLE
Update codeception.yml

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW 

# Summary
- Adds shuffle to the mix.

# Review By (Date)
- Whenever

# Urgency
- Low

### CODE REVIEW

File: config/codeception/codeception.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ paths:
   support: tests/_support
   envs: tests/_envs
 actor_suffix: Tester
+settings:
+  shuffle: true
```
Comment (pookmish) [inline config/codeception/codeception.yml:9]: what dis? it shuffles the order of the tests?

Comment (sherakama) [reply config/codeception/codeception.yml:9]: Yeah, so that we ensure we don't have a dependency on order of operations on the tests.

Comment (sherakama) [reply config/codeception/codeception.yml:9]: https://codeception.com/docs/07-AdvancedUsage#shuffle

Comment (sherakama) [reply config/codeception/codeception.yml:9]: I use it locally for maximum hilarity.

Comment (pookmish) [reply config/codeception/codeception.yml:9]: thats cool. i like it.

**Review by pookmish — APPROVED** [review]

**Pull request merged**